Week1/exercise/app.js

```diff
@@ -0,0 +1,77 @@
+'use strict';
```

Comment:
This exercise file is not part of the homework and should not be in your pull request.